src/AddTodoForm.js

```diff
@@ -1,13 +1,35 @@
 import React from "react";
+import { useState } from "react";
 
-function AddTodoForm() {
+const AddTodoForm = ({onAddTodo}) => {
+    const [todoTitle, setTodoTitle] = useState('');
+    const handleTitleChange = (e) => {
+        const newTodoTitle = e.target.value
+        setTodoTitle(newTodoTitle)
+    }
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        console.log(todoTitle);
+        onAddTodo({title: todoTitle}, {id: Date.now()});
+        event.target.title.value = ''    
```

Comment:
The value reset isn't working : (. You'll have to use your new state setting function!